### ISSUE TITLE
Input field without a label

### ISSUE THREAD

**Issue by selikem** [soniasophia/aloha-website#2]
Input elements should have label elements associated with them to aid accessibility. One reason for this is to provide a screen reader with a descriptive text to indicate what the input field requires.
